doc/tech/README.md

```diff
@@ -3,203 +3,170 @@
-
-There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualization:
-
-- Para-Virtualization method (software only).
-- Direct Assignment via h/w assisted PCI-Passthrough (IOMMU).
-- Device Sharing with SR-IOV & h/w assisted PCI-Passthrough (IOMMU).
-- Para-Virtualization method with Hardware support.
+<a name="1.1"></a>
+## 1.1 Glossary
+The definition and intent of the terminology used throughout the documents is defined in the [Glossary](./glossary.md).
 
-**Figure 3** below shows some of the relevant IO Virtualization techniques.
+<a name="1.2"></a>
+## 1.2 Problem Statement
+Based on informal conversations with many operators and developers, there is a realization that there are significant technical, operational and business challenges to the development and deployment of VNF applications related to the lack of a common virtualized infrastructure platform.  These include but are not limited to the following:
 
-<p align="center"><img src="./figures/tech_iov.png" alt="scope" title="Document Types" width="70%"/></p>
-<p align="center"><b>Figure 3:</b> Relevant IO Virtualization Techniques</p>
+ - Higher development costs due to the need to develop Virtual Network Functions (VNF) on multiple custom platforms for each operator
+ - Increased complexities due to the need to maintain multiple versions of applications to support each custom environemnt
+ - Lack of Testing and validation commonalities, leading to inefficiencies and increased time to market. While the operators will still do internal testing, but using an industry driven verification program based on a common NFVI would provide a head start.
+ - Slower adoption of cloud-native NFV applications and architectures.  A Common Telco NFVI may provide an easier path to methodologies that will drive faster cloud-native NFV application development.
+ - Increased operational overhead due to the need for operators to integrate diverse and sometime conflicting VNF platform requirements.
 
-<a name="4.1.1"></a>
-#### 4.1.1 Para-virtualization method (software only)
+One of major challenges holding back the more rapid and widespread adoption of VNF is that the traditional telecom ecosystem vendors, while building or designing their virtualized services (whether it be Voice over LTE (VoLTE), Evolved Packet Core (EPC), or popular customer facing enterprise services such as SD WAN (Software Defined Wide Area Network), are making their own infrastructure assumptions and requirements, often with custom design parameters. This leaves the operators being forced to build complex integrations of various vendor/function specific silos which are incompatible with each other and might possibly have different and conflicting operating models. In addition, this makes the onboarding and certification processes of VNFs (coming from different vendors) hard to automate and standardise.  
```

Comment:
virtualized -> virtualised

Reply:
fixed in: https://github.com/cntt-n/CNTT/pull/1267/commits/3be9fda4e4d5e83d3bb6f1d21717f37991197006